server/static/css/oauth.css

```diff
@@ -1,15 +1,28 @@
 .oauth {
-    text-align: left;
-    padding-top: 0;
-    padding: 0;
-}
-.oauth-button-block {
-    padding-left: 20px;
-    padding-right: 20px;
     padding-bottom: 15px;
 }
-.oauth-allow-btn {
+.container {
+  min-height: 100%;
+  /* equal to footer height */
+  margin: 0 auto -70px;
```

Comment:
I moved this from landing because there's enough content to not need to bring the footer up.